.github/workflows/release.yml

```diff
@@ -2,33 +2,33 @@ name: Release
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
-      - 'v[0-9]+.[0-9]+.[0-9]-[a-zA-Z0-9]+'
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]-[a-zA-Z0-9]+"
 
 jobs:
   dist:
     strategy:
       matrix:
-        go: ['1.19.x']
+        go: ["1.19.x"]
```

Comment:
given that our `Dockerfile` is now `1.20.4`, do we want to swap this release strategy to be `1.20.x`?